cmd/obi-schema/main.go

```diff
@@ -0,0 +1,363 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I wonder if this belongs to this repo, as it looks pretty generic. Are there any project ins the open-telemetry ecosystem doing that? Maybe its worth to move this somewhere later if that can be reused?